packages/webapp/src/components/ui/Persona/index.tsx

```diff
@@ -31,7 +34,11 @@ export const Persona: StandardFC = memo(function Persona({ className }) {
 			>
 				{/* TODO: remove stack in favor of styled div component */}
 				<div className='d-flex align-items-center justify-content-center'>
-					<div className='pr-3 me-3'>{c('personaTitle', { firstName })}</div>
+					{isLG && (
+						<div className={cx(style.userName, 'pr-3', 'me-3')} title={`${firstName} ${lastName}`}>
```

Comment:
You can remove the bootstrap classes here and in L33 into the SCSS file, then you can remove the `cx` dependency as well.

Even better, do we need this `<div>` can we simply display the `firstname`?

Reply:
We use the cx mix in another spot in this file, so I'll have to change that too. However, I can cut the greeting string, yes.

Reply:
Rather, I need that div for the style and title tooltip for very long names at least, but I'll condense the bootstrap style info into that class to avoid a cx.